backend/app/services/image_processor.py

```
@@ -0,0 +1,122 @@
+    clean_image = Image.new(image.mode, image.size)
+    clean_image.putdata(list(image.getdata()))
+    return clean_image
+
+
+def validate_magic_bytes(file_content: bytes):
+    """
+    Validates the actual file type of image
+    """
+    fmt = imghdr.what(None, h=file_content)
+    if fmt not in ["jpeg", "png", "webp"]:
+        raise ValueError("Invalid or unsupported image type.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)